static/app/types/reflux.d.ts

```diff
@@ -0,0 +1,8 @@
+import {SafeRefluxStore, SafeStoreDefinition} from 'sentry/utils/makeSafeRefluxStore';
+import {Store, StoreDefinition} from 'reflux';
+
+declare module 'reflux' {
+  function createStore<T extends SafeStoreDefinition | StoreDefinition>(
```

Comment:
I'm happy w/ this.

It does expose some internals right? I don't think I care though

Reply:
Yeah, same thing. If you check the final type it's pretty much the same + it'll add the safe store definition if you are passing through that helper
<img width="640" alt="CleanShot 2022-04-01 at 15 35 21@2x" src="https://user-images.githubusercontent.com/9317857/161330100-0750270f-bca9-4010-8a05-41568218a78a.png">

Reply:
You'll end up being able to retrieve some internal values though now right. Like if it had a `Internal` `state` you can do `.state` now on the store 🤷‍♂️

Reply:
Yeah, for sure. CommiterStoreDefinition extends Internal and CommonStoreDefinition